library/core/src/cmp.rs

```diff
@@ -441,6 +441,44 @@ impl Ordering {
             _ => self,
         }
     }
+
+    /// Checks if the `Ordering` is an `Equal` or `Greater` variant.
+    ///
+    /// Returns `true` if so, `false` otherwise.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use std::cmp::Ordering;
+    ///
+    /// assert_eq!(Ordering::Less.is_equal_or_greater(), false);
+    /// assert_eq!(Ordering::Equal.is_equal_or_greater(), true);
+    /// assert_eq!(Ordering::Greater.is_equal_or_greater(), true);
+    /// ```
+    #[inline]
+    #[must_use]
+    pub const fn is_equal_or_greater(self) -> bool {
```

Comment:
nit: given that it's `PartialOrd::ge`, consider `is_greater_or_equal`.

Reply:
It's now `is_ge`

Reply:
Oh, it didn't already have `is_equal` and such for single variants.  `is_ge` sounds much better than my suggestion, then.